util/design/lib/common.py

```diff
@@ -2,12 +2,15 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 r'''Common utilities used by various util/design scripts.'''
-
-import random
+import os
+import sys
 import re
 import textwrap
 from math import ceil, log2
 from pathlib import Path
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../'))
```

Comment:
you should be able to use the same `imports = ["../../"]` trick here too in the BUILD file I think, then you can eliminate this too